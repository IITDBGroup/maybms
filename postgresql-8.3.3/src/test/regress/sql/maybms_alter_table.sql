--test for altering uncertain table
create table s(a int, b int);

create table r as repair key a in s;

alter table r add column c int;

alter table r drop column a;

alter table r drop column _v0;

alter table r drop column _d0;

alter table r drop column _p0;

drop table r;
drop table s;



