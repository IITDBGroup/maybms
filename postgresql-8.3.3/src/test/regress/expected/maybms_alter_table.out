--test for altering uncertain table
create table s(a int, b int);
create table r as repair key a in s;
alter table r add column c int;
ERROR:  Query not supported: ALTER TABLE not supported on uncertain relations.
alter table r drop column a;
ERROR:  Query not supported: ALTER TABLE not supported on uncertain relations.
alter table r drop column _v0;
ERROR:  Query not supported: ALTER TABLE not supported on uncertain relations.
alter table r drop column _d0;
ERROR:  Query not supported: ALTER TABLE not supported on uncertain relations.
alter table r drop column _p0;
ERROR:  Query not supported: ALTER TABLE not supported on uncertain relations.
drop table r;
drop table s;
